src/ReportManager/DetailsSection/index.test.js

```diff
@@ -5,22 +5,24 @@ import userEvent from '@testing-library/user-event';
 import { createMapMock } from '../../__test-helpers/mocks';
 import { eventSchemas } from '../../__test-helpers/fixtures/event-schemas';
 import { eventTypes } from '../../__test-helpers/fixtures/event-types';
-import { GPS_FORMATS } from '../../utils/location';
 import { formValidator } from '../../utils/events';
+import { GPS_FORMATS } from '../../utils/location';
 import { MapContext } from '../../App';
-import MapDrawingToolsContextProvider, { MapDrawingToolsContext } from '../../MapDrawingTools/ContextProvider';
+import { MapDrawingToolsContext } from '../../MapDrawingTools/ContextProvider';
 import { mockStore } from '../../__test-helpers/MockStore';
 import patrolTypes from '../../__test-helpers/fixtures/patrol-types';
-import { render, screen, waitFor, within } from '../../test-utils';
+import { render, screen, within } from '../../test-utils';
 import { report } from '../../__test-helpers/fixtures/reports';
 import { VALID_EVENT_GEOMETRY_TYPES } from '../../constants';
 
 import DetailsSection from './';
 
 describe('ReportManager - DetailsSection', () => {
-  const onFormChange = jest.fn(),
```

Comment:
**Related to AJV validations**
Had to redo (and added a few missing) most tests after the changes.